grudge/reductions.py

```diff
@@ -268,11 +268,17 @@ def integral(dcoll: DiscretizationCollection, dd, vec) -> "DeviceScalar":
 def _apply_elementwise_reduction(
         op_name: str, dcoll: DiscretizationCollection,
         *args) -> ArrayOrContainerT:
-    r"""Returns a vector of DOFs with all entries on each element set
-    to the reduction operation *op_name* over all degrees of freedom.
+    r"""Returns an array container whose entries contain
+    the elementwise reductions in each cell.
 
     May be called with ``(vec)`` or ``(dd, vec)``.
 
+    Note that for array contexts which support nonscalar broadcasting
+    (e.g. :class:`meshmode.array_context.PytatoPyOpenCLArrayContext`),
+    the size of each component vector will be of shape `(nelements, 1)`.
```

Comment:
Minor RST formatting:
```suggestion
    the size of each component vector will be of shape ``(nelements, 1)``.
```

Reply:
Thanks for catching this!